.github/workflows/test.yml

```diff
@@ -1,11 +1,43 @@
 name: Test application (e2e)
 
 on:
+  push:
+    tags:
+      - cypress
+
   pull_request:
     branches:
       - master
 
+
 jobs:
+  cypress-test:
+    runs-on: ubuntu-16.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Make env
+        run: |
+          cat > .env << EOF
+          ${{ secrets.ENVFILE }}
+          EOF
+
+      - name: Build Locally
+        run: make local-build
+
+      - name: Run locally
+        run: make local-run &
+
+      - name: Waiting for server to spin up
+        run: |
+          while ! nc -z localhost 4000; do
```

Comment:
I hope this doesn't put GH Actions in a state where it runs forever and eats into our compute quota. I guess `make local-run` would have to fail for that to be the case which would kill the Action.

Reply:
On second thought, Docker Compose doesn't fail if a service fails, so I think this is a possibility.

Reply:
Logging another ticket to look into this :)